### DISCUSSION TITLE
Browser bookmark icon for dark theme is wrong

### DISCUSSION THREAD

**Discussion by sanjarcode** [community/community#30613] [Accessibility]
![image](https://user-images.githubusercontent.com/48156230/186333235-befaa93e-2799-4996-aef2-35f483e50b5d.png)
The icon should have white outline with black inside. Example:
![image](https://user-images.githubusercontent.com/48156230/186333374-9a13ff15-bda4-41de-9436-fd85cfe41c08.png)

**Comment by radar-blip** [comment]
You can file a bug report if you like, here: 
[Bug-Report](https://support.github.com/contact/bug-report)

This might help get it fixed so that you can see it better! 👍🏻

**Comment by benjiallen** [comment]
@sanjarcode couple of questions to help reproduce the issue:

- What operating system and browser are you using?
- What GitHub theme are you using?
- Does this happen in your bookmarks bar?
- Does this happen in an active tab?

**Reply by sanjarcode** [reply]
> @sanjarcode couple of questions to help reproduce the issue:
> 
> - What operating system and browser are you using?
> - What GitHub theme are you using?
> - Does this happen in your bookmarks bar?
> - Does this happen in an active tab?

1.OS - Ubuntu 20.04, browser - Google Chrome
2. GitHub theme - Dark default
3. Yes, in the bookmarks bar
4. No effect if tab is active or not

**Reply by sanjarcode** [reply]
Steps to reproduce:
Step 1: Enable the dark theme. There are two ways to do this:
    1. Enable OS wide dark theme (Ubuntu.Settings > Appearance > Dark), and use set GTK+ as theme in Chrome.Settings > Appearance
    2. Install 'Just Black' theme (by Google, available in the Chrome extensions store) and set is as the browser theme.

Step 2: Bookmark any GitHub link to the bookmarks bar.
Done

**Comment by cehfisher** [comment]
Hello @sanjarcode - this issue seems to have been resolved. Could you please confirm and close this issue? Thanks! (cc @queenofcorgis)

**Comment by safhossain** [comment]
This has still not been resolved for Firefox LOL

**Reply by cehfisher** [reply]
Hello @safhossain - this is a very old thread and resolved for the original poster. If you are experiencing this issue on a different browser, could you please open a new one and include as many details as possible so the team can try to reproduce it? Thanks!
